app/modules/encounters/models.py

```diff
@@ -165,6 +165,23 @@ def get_taxonomy_guid(self, sighting_fallback=True):
             taxonomy_guid = self.sighting.get_taxonomy_guid()
         return taxonomy_guid
 
+    def get_taxonomy_names(self, **kwargs):
```

Comment:
Have a look in the Taxonomy class, doesn't get_all_names give you this?

Reply:
Thanks!  I'm going to look into it

Reply:
Does it look right?

```python
return Taxonomy(taxonomy_guid).get_all_names()
```

Reply:
Looks a lot simpler, as long as it does the same thing (which is what I was less certain about) then that's great

Reply:
Yeah, I wasn't sure either.  That's why I added a test to check the output is still the same